core/api/service/api_service.cpp

```diff
@@ -162,15 +166,16 @@ namespace kagome::api {
     logger_->debug("Service stopped");
   }
 
-  outcome::result<uint32_t> ApiService::subscribe_session_to_keys(
+  outcome::result<uint32_t> ApiService::subscribeSessionToKeys(
       const std::vector<common::Buffer> &keys) {
     if (auto session_id = threaded_info.fetchSessionId(); session_id) {
       if (auto session = findSessionById(*session_id)) {
+        const auto id = session->generate_subscription_set_id();
```

Comment:
This name violates the method naming convention. Please, rename in camel-case style.